### PR TITLE
Play 9c on x86 macOS / Linux using CI Artifacts (UNOFFICIAL SUPPORT)

### PR DESCRIPTION
**Pull request by Akamig**
Addresses #9. some outdated or internal only stuff yanked.

### CODE REVIEW

File: docs/how-to-play-9c-on-linux-and-mac.md [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+## 3. Click Artifact
+
+
+
+## 4. Download and Decompress Artifact
+
+
+
+# 2. Run on Linux
```
Comment (U-lis) [inline docs/how-to-play-9c-on-linux-and-mac.md]: I think `# 2` should be more general title and each OS guideline should belongs to the `# 2` section.

File: docs/how-to-play-9c-on-linux-and-mac.md [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+In case of permission issue, `chmod +x ./Nine Chronicles` will do the trick.
+
+
+Now it works!
+
+config.json file is
+    - (on macOS) `~/Library/Application Support/Nine Chronicles/config.json`
+    - (on Linux) `~/.config/Nine Chronicles/config.json`
+
+And you can import private key using Libplanet.Tools, so called [`planet`](https://github.com/planetarium/libplanet/releases). or, you can mount windows and copy keystore folder under `%APPDATA%/planetarium/keystore` and put it on `~/.config/planetarium/keystore` I'll trust you would work this part out by yourself.
```
Comment (U-lis) [inline docs/how-to-play-9c-on-linux-and-mac.md]: Please spread this data to appropriate os section.

File: docs/how-to-play-9c-on-linux-and-mac.md [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+# 2. Run on Intel Mac
+
+I don't have mac.
+
+But kind enough to left explanation.
```
Comment (U-lis) [inline docs/how-to-play-9c-on-linux-and-mac.md]: I don't know about our tone & manners of docs at this time :thinking: 
But I think joke-like expressions are used with care.
What do you think?

File: docs/how-to-play-9c-on-linux-and-mac.md [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+
+from System Preference → Click Open Anyway
+
+How Fascinating.
+
+
+## 3.1. TroubleShooting
+
+### 3.1.1. .NET core 3.1 installation on Ubuntu 22.04 (220729 @Hyeon)
```
Comment (U-lis) [inline docs/how-to-play-9c-on-linux-and-mac.md]: Please remove my name lol

**Review by U-lis — DISMISSED** [review]

File: docs/how-to-play-9c-on-linux-and-mac.md (hunk 4 above)
Comment (Akamig) [reply docs/how-to-play-9c-on-linux-and-mac.md]: OK

File: docs/how-to-play-9c-on-linux-and-mac.md (hunk 3 above)
Comment (Akamig) [reply docs/how-to-play-9c-on-linux-and-mac.md]: I wrote this post originally and I want users to feel the same as before the translation.

File: docs/how-to-play-9c-on-linux-and-mac.md (hunk 2 above)
Comment (Akamig) [reply docs/how-to-play-9c-on-linux-and-mac.md]: OK!

File: docs/how-to-play-9c-on-linux-and-mac.md (hunk 1 above)
Comment (Akamig) [reply docs/how-to-play-9c-on-linux-and-mac.md]: Yes, you're right, actually I think this document's levels needs to be totally reordered. will be fixed.

File: docs/how-to-play-9c-on-linux-and-mac.md (hunk 3 above)
Comment (Akamig) [reply docs/how-to-play-9c-on-linux-and-mac.md]: ...but I'll change it to more formal tone.

**Review by U-lis — APPROVED** [review]

**Review by U-lis — APPROVED** [review]
Some of images make dizziness but I know those images are recycled. Be aware that the readers may want to change to more calm(lol) image.

**Pull request merged**